src/main/java/com/alkemy/ong/controller/v1/AuthController.java

```diff
@@ -32,5 +38,10 @@
 
     }
 
+    @GetMapping("/me")
+    public ResponseEntity<User> userLogged()throws UserNotFoundException {
```

Comment:
It is a bad practice to use entities in the Controller, please return a dto instead.